### ISSUE TITLE
Docs:Confusion on necessity of logging service

### ISSUE THREAD

**Issue by devinrhode2** [damiencaselli/ember-cli-sentry#114]
In the readme, it says the only required property is the dsn, but it appears that you actually do need to include a serviceName and exposedPropertyName, and most people are also going to want `development: environment === 'development',` and the cdn property. 

It would've been nice there was simply a recommended "works for most" config in the readme.

I added "(globalErrorCatching is set to true by default)" to the cdn block comment, but otherwise this is the config that works for me:
```
  ENV.sentry = {
    /**
     * If Raven.js is not inlined in your code, this is
     * where you want to define it.
     *
     * I recommend **not** using a build containing ember.js plugin
     * (https://github.com/getsentry/raven-js/blob/master/plugins/ember.js)
     * since setting `globalErrorCatching: true` will perform the same
     * operation, but safely. (globalErrorCatching is set to true by default)
     *
     * @type {String}
     * @default undefined
     */
    cdn: 'https://cdn.ravenjs.com/3.3.0/raven.min.js',

    /**
     * The only mandatory parameter.
     *
     * @type {String}
     */
    dsn: 'https://8d56380642f14be19b6c6f076cfbce6d@sentry.io/271452',

    /**
     * If set to true, it will prevent Raven.js from being initialized.
     * Errors and logs will be logged to the console (default) instead of
     * being reported by Raven.
     *
     * @type {Boolean}
     * @default undefined
     */
    development: environment === 'development',

    /**
     * Injects the logging service as this property.
     *
     * @type {String}
     * @default 'raven'
     */
    exposedPropertyName: 'logr',

    /**
     * Service used to interface with Raven.
     *
     * @type {String}
     * @default 'raven'
     */
    serviceName: 'logr'
  };
```

**Comment by devinrhode2** [comment]
Another thing I found very confusing, was whether or not I actually did need to generate a logging service. I know at the top of the readme is says "addon does not generate the logging service", but other statements confused me and made me question whether or not I really do need to generate a logging service.

In the install section,
```
The addon will assume there is an available service that proxies Raven, which is not the case unless you already did the install. (OK I just did the `ember install` - so I must not need to generate the logging service?)
//Better wording:
replace: "..Raven, which is not the case unless you already did the install."
with: "..Raven."
or: "..Raven, which is not the case unless this addon has been setup in the past."

The easiest way of doing it is to create a service only extending ember-cli-sentry/services/raven:

aboutraw—
// your-app/services/custom-logger.js
export { default } from 'ember-cli-sentry/services/raven';
You can also use a generator ember g logger <logger-name>, which will generate a service called <logger-name> extending ember-cli-sentry/services/raven and exposing its methods and properties.
If you create the logger service that way, note that you must use the same name for the serviceName property. (BUT WAIT - so if I create a file named "custom-logger.js" ... I don't need to set the serviceName to "custom-logger"??? Again - maybe since I don't actually have to create a customer logger?)
```

Then the other confusing thing is that there IS a serviceName and exposedPropertyName set to something that seems like it should "Just Work" - and it doesn't match the name of the logger file name you suggest above, "custom-logger"

**Comment by devinrhode2** [comment]
The only required parameter is `dsn` BUT it's clear that this addon *does not generate the logging service for you* - and this makes it sound like that is something users need to do (there's plenty of conflicting info here around whether or not you need to generate the logging service)

**Comment by devinrhode2** [comment]
Maybe could just add at the end of "Does not generate logging service" you can add " (optional)", unless it's actually NOT optional...

**Comment by buschtoens** [comment]
Also tripped over this. As a newbie to sentry and this addon, I did not know what

> This addon does **not**:
> * Generate the logging service for you.

is supposed to mean. And neither the **Install** or **Configuration** section explain how to create the service. Since #99 remove the `logger` blueprint, I recommend adding one or two sentences to the README.

**Comment by duizendnegen** [comment]
Welcoming a PR with updated docs 👍

**Comment by devinrhode2** [comment]
we need someone to address our points of confusion..

**Comment by Turbo87** [comment]
@devinrhode2 the README on the main project page is for the `master` branch, where those properties are no longer needed. The latest released version is still v2.x though, so make sure to read the docs for the correct version. v3.0 will be released soon though and you can already use the beta version for that if you want to give it a try.

**Comment by dbinetti** [comment]
I'm afraid I'm still confused on how to use this with 3.x @Turbo87 .  As it stands I've installed the 3.0.0 beta three, and set the DSN config.  I can see the raven service in the ember inspector container, but nothing I try triggers and actual exception in Sentry.  is this the expected behavior?

It would be spectacular to know, step by step, what's entailed in getting this working.

**Comment by Turbo87** [comment]
@dbinetti did you try this in production or locally on the dev server? the addon will only actually send something to the sentry servers if it's running in a production environment

**Comment by dbinetti** [comment]
Both. Worked in niether

**Comment by Turbo87** [comment]
you can use `require('raven').default.isSetup()` to check if the sentry lib itself was setup correctly. if that is the case you can use the Ember Inspector to get a reference (usually `$E`) to the `raven` service in the container and then use `$E.captureMessage('test')` to send a test message. have a look at the network tab of the dev tools to check if the message was sent correctly to the sentry servers.

**Comment by dbinetti** [comment]
Thank you, that helped me diagnose.

As it turns out this is a epic facepalm -- I have two sentry instances, one
for front-end and one for back-end -- and I put in the wrong DSN.  So all
the events were going to the wrong place.  Sigh.

Thanks for your help

**Comment by Turbo87** [comment]
I just released `3.0.0`, so this issue should be resolved now.
